pkg/packer/manifests/vsphere/base/packer-base.json

```diff
@@ -9,7 +9,7 @@
         "export_manifest": "none",
         "folder": "",
         "guest_os_type": null,
-        "ib_version": "{{env `IB_VERSION`}}",
+        "kib_version": "{{env `KIB_VERSION`}}",
         "username": "{{env `VSPHERE_USERNAME`}}",
```

Comment:
building base templates from kib is deprecated and moved to https://github.com/mesosphere/vsphere-base-template .
I will file separate PR to remove these files.